src/main/java/com/scottlogic/competition_engine/application/TestController.java

```diff
@@ -0,0 +1,18 @@
+package com.scottlogic.competition_engine.application;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TestController {
+
+    @Autowired
+    public TestController() {}
+
+    @RequestMapping(value = "/hello")
```

Comment:
I would make this "@GetMapping" instead. Makes it more clear that this a get request. From what I can tell RequestMapping acts as a catch-all annotation